client/src/frontend/components/journal/IssueList/styles.js

```diff
@@ -13,3 +14,10 @@ export const List = styled.div`
   flex-direction: column;
   gap: 60px;
 `;
+
+export const LinkWrapper = styled(Box)`
+  > div {
+    background-color: var(--color-base-neutral-white);
+    padding: 0;
+  }
+`;
```

Comment:
This is bad but I couldn't come up with another way of getting the link in the right place, aside from refactoring this whole view into an `EntityCollection` pattern. Should we do that?